css/mediaqueries/custom-003.html

```diff
@@ -0,0 +1,46 @@
+<!doctype html>
+<html>
+	<head>
+		<title>Test: custom media queries that are undefined behave different from queries that evaluate to false</title>
+		<link rel="author" title="Romain Menke" href="https://github.com/romainmenke">
+		<link rel="help" href="https://www.w3.org/TR/mediaqueries-5/#at-ruledef-custom-media">
```

Comment:
Maybe also link to both https://www.w3.org/TR/mediaqueries-4/#evaluating and https://www.w3.org/TR/mediaqueries-4/#error-handling for this test?

Reply:
Good suggestion!
I've added these links.